content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -53,15 +55,16 @@ takes you through this example in more detail).
 ### Cron schedule syntax
 
 ```
-# ┌───────────── minute (0 - 59)
-# │ ┌───────────── hour (0 - 23)
-# │ │ ┌───────────── day of the month (1 - 31)
-# │ │ │ ┌───────────── month (1 - 12)
-# │ │ │ │ ┌───────────── day of the week (0 - 6) (Sunday to Saturday;
-# │ │ │ │ │                                   7 is also Sunday on some systems)
-# │ │ │ │ │
-# │ │ │ │ │
-# * * * * *
+#   ┌──────────────── timezone (optional)
+#   |    ┌───────────── minute (0 - 59)
+#   |    │ ┌───────────── hour (0 - 23)
+#   |    │ │ ┌───────────── day of the month (1 - 31)
+#   |    │ │ │ ┌───────────── month (1 - 12)
+#   |    │ │ │ │ ┌───────────── day of the week (0 - 6) (Sunday to Saturday;
+#   |    │ │ │ │ │                                   7 is also Sunday on some systems)
+#   |    │ │ │ │ │
+#   |    │ │ │ │ │
+# TZ=UTC * * * * *
```

Comment:
It looks like we're now using the format from https://pkg.go.dev/github.com/robfig/cron/v3#hdr-CRON_Expression_Format :sob:

The documented timezone variable there is `CRON_TZ`; `TZ` is listed as a legacy variable.

Reply:
Thank you very much, I'll update it